src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
```

Comment:
```suggestion
const BASE_API_URL = 'https://mate-academy.github.io/fe-students-api';
```

Please have a look at the task description which suggests using this [API](https://mate-academy.github.io/fe-students-api/posts.html)